Uno.Gallery/Uno.Gallery.Shared/App.xaml.cs

```diff
@@ -48,6 +49,7 @@ public App()
 #endif
 
 			InitializeLogging();
+			ConfigureFeatureFlags();
```

Comment:
I'm guessing we should wrap this in a `#if !WINDOWS` and there's also a feature flag set on like 48 that we can place in this new method as well